### ISSUE TITLE
Add Open API to Service a

### ISSUE THREAD

**Issue by Emily-Jiang** [eclipse/microprofile-service-mesh-service-a#25]
We should Open API to the service a and then demonstrate how it can be accessible from Istio service mesh.

**Comment by kenfinnigan** [comment]
With mTLS enabled the only way for the endpoint to be accessible is by adding an Ingress Gateway rule for it

**Comment by pilhuhn** [comment]
I don't think this is an issue at all, as with Istio you want to have all traffic go via the Ingress-GW and not directly via OpenShift route or plain Kubernetes Ingress.

If the traffic goes via IG, then we need to slightly set up the GW-VS differently to not only

match
 [ - uri:
      prefix: /mp-serv...](url)

but also
```
  - uri:
      exact: /openapi

```
and potentially

```
  - uri:
      exact: /metrics

```
to demo / expose this.
